frontend/src/pages/AnnouncementPage.js

```diff
@@ -119,88 +130,163 @@ function AnnouncementEntry({
 		WebkitLineClamp: 2,
 	} : {};
 
-	const boxSidebarStyle = { display: "flex", mt: 1, mb: 1, pb: 1, borderBottom: 1, borderColor: "grey.800" };
-	const boxPageStyle = { display: "flex", mt: 1, mb: 3, p: 1, border: 1, borderColor: "grey.800" };
+	const boxPageStyle = { display: "flex", mt: 1, mb: 3, p: 1, border: 1, borderColor: "rgba(255, 255, 255, 0.1)" };
 	const titleStyle = { fontFamily: "Montserrat", fontSize: "14px", fontStyle: "normal", fontWeight: "bold" };
-	const dateStyle = { 
-		fontFamily: "Noto Sans", fontSize: "11px", fontStyle: "normal", fontWeight: "medium", marginBottom: 3, opacity: 0.5
+	const dateStyle = {
+		fontFamily: "Noto Sans", fontSize: "11px", fontStyle: "normal",
+		fontWeight: "medium", opacity: 0.4
 	};
-	const contentStyle = { fontFamily: "Noto Sans", fontSize: "12px", fontStyle: "normal", fontWeight: "medium" };
+	const contentStyle = {
+		fontFamily: "Noto Sans", fontSize: "12px",
+		fontStyle: "normal", fontWeight: "medium",
+		color: "rgba(255, 255, 255, 0.81)"
+	};
+	const randomPhoto = Photos[Math.floor(Math.random() * Photos.length)];
 
-	return <div style={{ width: "100%" }}>
-		<Box xs={ 12 } sx={ isSidebar ? boxSidebarStyle : boxPageStyle }>
-			<div style={{ width: "100%", padding: 5 }}>
-				<Typography style={ titleStyle } sx={ limitLines } variant="h5">
-					{announcement.title}
-				</Typography>
-				<Typography style={ dateStyle } sx={{ mb: 1 }}>{formated_created_at}</Typography>
-				<Typography style={ contentStyle } sx={{ mb: 1 }, limitLines}>
-					{announcement.content}
-					{/* <Link href="/announcements" color="inherit">Read more</Link> */}
-				</Typography >
-				{userIsAdmin
-					?
-					<div style={{ textAlign: "center", marginTop: 3 }}>
-						<Button onClick={handleDeleteButton(announcement.$id)} variant="outlined" sx={{ m: 1 }}>
-							Delete
-						</Button>
-						{isSidebar
-							?
-							<Button
-								component={RouterLink} to={"/announcements#" + announcement.$id}
-								variant="outlined" sx={{ m: 1 }}
-							>
-								Edit
+	const sidebarComponents =
+		<div style={{ mb: 2 }}>
+			<Box>
+				<Grid container spacing={0} sx={{ maxWidth: "sm" }}>
+					<Grid item xs={9}>
+						<Link href="/announcements" style={{}}>
+							<div
+								style={{
+									backgroundImage: `url(${randomPhoto})`,
+									backgroundSize: "100%",
+									backgroundRepeat: "no-repeat",
+									backgroundPosition: "center",
+									width: "126px",
+									height: "100px",
+								}}
+							/>
+						</Link>
+					</Grid>
+					<Grid
+						item xs
+						sx={{
+							height: "112px", borderBottom: 1, pl: 1,
+							borderColor: "rgba(255, 255, 255, 0.1)", mb: 2
+						}}
+					>
+						<Typography style={titleStyle} sx={limitLines} variant="h5">
```

Comment:
"Typography" is not right. It needs to change to HeaderTypography and ParagraphTypography respectively.